e2e/e2e_test.go

```diff
@@ -18,9 +18,7 @@ package e2e
 
 import (
```

Comment:
`fromHederaTransactionID` can be placed in the helpers once the other PR is merged -> `helpers/hedera/transaction-id.go`